R/run-model.R

```diff
@@ -328,7 +328,7 @@ naomi_prepare_data <- function(data, options) {
 
   if(is.null(options$calendar_quarter_t5)) {
     # If T5 is not specified, set T4 to 36-months after T3
-    t5 <- calendar_quarter_to_quarter_id(calendar_quarter_t3) + 9
+    t4 <- calendar_quarter_to_quarter_id(calendar_quarter_t4) + 3
```

Comment:
Thanks — I think you have an error here though; should `t4 <- ...` be `t5 <- ...`?

Also, does the test catch this?